newapi/ooniapi/private.py

```diff
@@ -769,26 +1046,36 @@ def api_private_country_overview():
     # TODO: add circumvention_tools_blocked im_apps_blocked
     # middlebox_detected_networks websites_confirmed_blocked
     probe_cc = validate_probe_cc_query_param()
-    # OLAP-use-case
-    s = sql.text(
-        """SELECT
-        MIN(measurement_start_day) AS first_bucket_date,
-        SUM(measurement_count) AS measurement_count,
-        COUNT(DISTINCT probe_asn) AS network_count
-        FROM counters_asn_noinput
-        WHERE probe_cc = :probe_cc
-    """
-    )
-    q = current_app.db_session.execute(s, {"probe_cc": probe_cc})
-    r = q.fetchone()
-    # NOTE: we are hardcoding a fixed first_bucket_date
+    if current_app.config["USE_CLICKHOUSE"]:
+        s = """SELECT
+            toDate(MIN(measurement_start_time)) AS first_bucket_date,
+            COUNT() AS measurement_count,
+            COUNT(DISTINCT probe_asn) AS network_count
+            FROM fastpath
+            WHERE probe_cc = :probe_cc
+            AND measurement_start_time > '2016-01-01'
```

Comment:
Why is a limit on 2016-01-01 being set here?

Reply:
Fixed, thanks.